### PR TITLE
[Wallet]: fix comments

### PR DESCRIPTION
**Pull request by plam-ml**
## Description 

- Fixing comments from https://github.com/MystenLabs/sui/pull/14059 and https://github.com/MystenLabs/sui/pull/14007

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx [hunk 1]
```diff
@@ -136,9 +119,23 @@ export function TokenRow({
 
 					{renderActions ? (
 						<div className="flex gap-2.5 items-center">
-							<TokenRowButton coinBalance={coinBalance} page="send" copy="Send" />
+							<TokenRowButton
+								coinBalance={coinBalance}
+								copy="Send"
```
Comment (Jordan-Mysten) [inline apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx:124]: `copy` should be `children`

**Review by Jordan-Mysten — CHANGES_REQUESTED** [review]
We're missing a resolution on these comments, either here as fixes, or on the originals PRs as explanations:

- https://github.com/MystenLabs/sui/pull/14059#discussion_r1343514148
- https://github.com/MystenLabs/sui/pull/14059#discussion_r1343514644
- https://github.com/MystenLabs/sui/pull/14007#discussion_r1343526222
- https://github.com/MystenLabs/sui/pull/14007#discussion_r1343525502
- https://github.com/MystenLabs/sui/pull/14007#discussion_r1343522841
- https://github.com/MystenLabs/sui/pull/14007#discussion_r1343521901
- https://github.com/MystenLabs/sui/pull/14007#discussion_r1343525208

**Comment by plam-ml** [comment]
https://github.com/MystenLabs/sui/pull/14007#discussion_r1343526222

This is because there is not a pool for `USDC > USDC`, it is assumed to be `1` so we can return early

**Review by pchrysochoidis — APPROVED** [review]

**Comment by plam-ml** [comment]
Putting left-over concerns on this ticket so it does not block this merge

**Pull request merged**